tasks/setup.yml

```
@@ -1,11 +1,11 @@
 ---
 - name: Add Cloudflare key
   apt_key:
-    url: 'https://pkg.cloudflare.com/pubkey.gpg'
+    url: 'https://pkg.cloudflare.com/cloudflare-main.gpg'
 
 - name: Add Cloudflare PPA
   apt_repository:
-    repo: 'deb http://pkg.cloudflare.com/ xenial main'
+    repo: 'deb http://pkg.cloudflare.com/cloudflared xenial main'
```

Question: Should we change `xenial` to something else for ubuntu 20.04?

---

Oh right, so something like this:

```suggestion
    repo: 'deb http://pkg.cloudflare.com/cloudflared bullseye main'
```

I'm not sure if more recent versions of Debian/Ubuntu works as intended while asking for a specific release version. I'm still new to Linux, any thoughts on this?